src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -0,0 +1,176 @@
+Default region name [None]: ....
+Default output format [None]: text
+```
+
+For more details, follow this link:
+
+> https://docs.openshift.com/container-platform/4.4/installing/installing_aws/installing-aws-account.html
+
+In this step we shall modify the default Cluster/Service CIDRs and deploy cluster-b on AWS.
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.132.0.0/14 |172.31.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-b
+```
+
+Change the Pod IP network. Please note it’s a /14 range by default so you need to use
++4 increments for “128”, for example: 10.132.0.0, 10.136.0.0, 10.140.0.0, ...
+ 
+```bash
+sed -i 's/10.128.0.0/10.132.0.0/g' cluster-b/install-config.yaml
+```
+
+Change the Service IP network. This is a /16 range by default, so just use +1 increments
+for “30”: for example: 172.31.0.0, 172.32.0.0, 172.33.0.0, ...
+
+```bash
+sed -i 's/172.30.0.0/172.31.0.0/g' cluster-b/install-config.yaml
+```
+
+And finally deploy
+
+```bash
+openshift-install create cluster --dir cluster-b
+```
+
+### Make your AWS cluster ready for Submariner
+
+Submariner gateway nodes need to be able to accept IPsec traffic. The default ports are 4500/UDP and 500/UDP.
+However, when you have some on-premise clusters (like vSphere in this example) which are typically inside a corporate network, the firewall configuration on the corporate router may not allow the default IPsec traffic.
+We can overcome this limitation by using non-standard ports like 4501/UDP and 501/UDP.
+
+Additionally, the default OpenShift deployments do not allow assigning an elastic public IP
+to existing worker nodes, something that's necessary at least on one end of the IPsec connections. 
+
+To handle these requirements on AWS, we provide a script that will prepare your AWS OpenShift deployment
+for Submariner, and will create an additional gateway node with an external IP.
+
+```bash
+
+curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
+chmod a+x ./prep_for_subm.sh
+
+./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+
+```
+
+> **_INFO_** Please note that  **oc**, **aws-cli**, **terraform**, and **unzip** need to be installed before running the `prep_for_subm.sh` script.
+
+Currently, prep_for_subm.sh script does not support specifying custom IPsec ports.
+Until the [issue](https://github.com/submariner-io/submariner/issues/240) is resolved, execute the following commands to open the necessary ports:
+
+```bash
+
+export BROKER_IKEPORT=501
+export NAT_PORT=4501
+sed "s/\ 500/\ $BROKER_IKEPORT/g" -i cluster-b/ocp-ipi-aws/ocp-ipi-aws-prep/ec2-resources.tf
+sed "s/\ 4500/\ $NAT_PORT/g" -i cluster-b/ocp-ipi-aws/ocp-ipi-aws-prep/ec2-resources.tf
+
+./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+
+```
+
+### Submariner Installation
+
+{{< subctl-install >}}
+
+#### Install Submariner with Service Discovery
```

Comment:
Can we also use include files for the replicated sections?

Reply:
Right, this is duplicated across couple of quickstart guides. Are you ok if we handle this in a separate PR?

Reply:
:+1: